src/app/home/index.tsx

```
@@ -48,6 +57,85 @@
   const { open, close } = useModal();
   const navigate = useNavigate();
 
+  const { data: totalView, refetch } = useApiTotalView();
```

ESLint

[error] 'refetch' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)